src/Jellyfin.Drawing/ImageProcessor.cs

```diff
@@ -403,10 +404,34 @@ public string GetImageBlurHash(string path, ImageDimensions imageDimensions)
         return _imageEncoder.GetImageBlurHash(xComp, yComp, path);
     }
 
+    /// <inheritdoc />
+    public string GetImageCacheTag(string baseItemPath, DateTime imageDateModified)
+        => (baseItemPath + imageDateModified.Ticks).GetMD5().ToString("N", CultureInfo.InvariantCulture);
+
     /// <inheritdoc />
     public string GetImageCacheTag(BaseItem item, ItemImageInfo image)
         => (item.Path + image.DateModified.Ticks).GetMD5().ToString("N", CultureInfo.InvariantCulture);
 
+    /// <inheritdoc />
+    public string GetImageCacheTag(BaseItemDto item, ItemImageInfo image)
+        => (item.Path + image.DateModified.Ticks).GetMD5().ToString("N", CultureInfo.InvariantCulture);
+
+    /// <inheritdoc />
+    public string? GetImageCacheTag(BaseItemDto item, ChapterInfo chapter)
+    {
+        if (chapter.ImagePath is null)
+        {
+            return null;
+        }
+
+        return GetImageCacheTag(item, new ItemImageInfo
```

Comment:
In this specific case you are only using the DateModified prop to compute the cache tag. It would be better with a (private?) overload that takes a string and a long, which is used by all 4.

Reply:
i agree its not perfect but i hasitate to refactor even more code in this PR.

Reply:
I don't consider that much of a refactor though.